price_feeds/solana/send_usd/app/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "send_usd",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@pythnetwork/price-service-client": "^1.8.2",
+    "@pythnetwork/pyth-solana-receiver": "^0.7.0",
+    "@solana/wallet-adapter-base": "^0.9.23",
+    "@solana/wallet-adapter-react": "^0.15.35",
+    "@solana/wallet-adapter-react-ui": "^0.9.35",
+    "@solana/wallet-adapter-wallets": "^0.19.10",
+    "@solana/web3.js": "^1.90.0",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.18.82",
+    "@types/react": "^18.2.0",
+    "@types/react-dom": "^18.2.19",
+    "buffer": "^6.0.3",
+    "craco": "^0.0.3",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "typescript": "^4.9.5",
+    "web-vitals": "^2.1.4",
+    "rpc-websockets": "7.11.0"
```

Comment:
got a build error `Error: Cannot find module 'rpc-websockets/dist/lib/client'` so explicitly declared the version here as suggested [here](https://stackoverflow.com/questions/78566652/solana-web3-js-cannot-find-module-rpc-websockets-dist-lib-client)